static/gsApp/views/amCheckout/checkoutSuccess.tsx

```diff
@@ -0,0 +1,688 @@
+  const creditApplied = utils.getCreditApplied({
+    creditApplied: data?.creditApplied ?? 0,
+    invoiceItems,
+  });
+  const total = isImmediateCharge
+    ? (invoice.amountBilled ?? invoice.amount)
+    : (previewData?.billedAmount ?? 0);
+  const commonChangesProps = {
+    plan: basePlan,
+    planItem,
+    reservedVolume,
+    products,
+    fees,
+    creditApplied,
+    total,
+    credits,
+  };
+
+  const contentTitle = isImmediateCharge
+    ? t('Pleasure doing business with you')
+    : effectiveToday
+      ? t('Consider it done')
+      : t('Consider it done (soon)');
+  const contentDescription = isImmediateCharge
+    ? tct(
+        "We've processed your payment and updated your subscription. Your plan will renew on [date].",
+        {date: renewalDate}
+      )
+    : effectiveToday
+      ? t("We've updated your subscription.")
+      : tct('No charges today. Your subscription will update on [date].', {
+          date: effectiveDate,
+        });
+
+  return (
+    <Content>
+      <InnerContent>
+        <Title>{contentTitle}</Title>
+        <Description>{contentDescription}</Description>
+        <ButtonContainer>
+          <LinkButton
+            aria-label={t('Edit plan')}
+            to="/settings/billing/checkout/?referrer=checkout_success"
+          >
+            {t('Edit plan')}
+          </LinkButton>
+          <LinkButton
+            priority="primary"
+            aria-label={t('View your subscription')}
+            to={`/settings/billing/overview/${viewSubscriptionQueryParams}`}
+          >
+            {t('View your subscription')}
+          </LinkButton>
+        </ButtonContainer>
+      </InnerContent>
+      {isImmediateCharge ? (
+        <Receipt
+          {...commonChangesProps}
+          charges={invoice.charges}
+          planItem={planItem as InvoiceItem}
+          dateCreated={invoice.dateCreated}
+        />
+      ) : effectiveToday ? null : (
+        previewData &&
+        effectiveDate && (
+          <ScheduledChanges {...commonChangesProps} effectiveDate={effectiveDate} />
+        )
+      )}
+    </Content>
+  );
+}
+
+export default CheckoutSuccess;
+
+const Content = styled('div')`
+  padding: ${p => p.theme.space['2xl']};
+  max-width: ${p => p.theme.breakpoints.xl};
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: auto 100px;
+  gap: ${p => p.theme.space['3xl']};
+
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    flex-direction: column;
+    gap: ${p => p.theme.space['3xl']};
+    margin: ${p => p.theme.space['3xl']};
+  }
+`;
+
+const InnerContent = styled('div')`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  max-width: 500px;
+  text-align: left;
+
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    align-items: center;
+    text-align: center;
+  }
+`;
+
+const Title = styled('h1')`
```

Comment:
@isabellaenriquez could you take a look at heading, text and container components here? I think you could remove a lot of the custom CSS from this page if you used them